src/palm-react-native-ui-kit/screens/nft/SendTokenScreen/TokenAmountInput.tsx

```diff
@@ -23,10 +24,15 @@ const TokenAmountInput = ({
   onSetValue?: Dispatch<SetStateAction<Token>>
   selectedNetwork: SupportedNetworkEnum
 }): ReactElement => {
+  const { t } = useTranslation()
+
   const tokenValue = useMemo(
     () =>
       value
-        ? UTIL.getTokenBalanceInUSD(UTIL.microfyP(value), item.price)
+        ? UTIL.getTokenBalanceInUSD(
+            UTIL.microfyP(UTIL.removeCommasToNumber(value) as Token),
+            item.price
+          )
         : undefined,
     [item, value]
   )
```

Comment:
can users even input comma? I thought it was only period. But if yes, then thank you

Reply:
just remove comma using regex

Reply:
There were already setComma, delComma functions

I removed this functions